Content.Server/GameObjects/Components/Interactable/HandheldLightComponent.cs

```diff
@@ -237,6 +237,10 @@ private void EjectCell(IEntity user)
                 cell.Owner.Transform.Coordinates = user.Transform.Coordinates;
             }
 
+            // Assuming the battery has just been taken out of the flashlight, make sure it's getting disabled
+            SetState(false);
+            Activated = false;
+
```

Comment:
Why not call `TurnOff`? If you don't want the sound, just add a `playSound` param to the func that defaults to true

Reply:
I've updated the code, what do you think of this?